### PR TITLE
ITNFA-22: added national person identifier type

### PR DESCRIPTION
**Pull request by pawelcieszko**

### CODE REVIEW

File: omod/src/main/resources/apps/isanteplus_registration_app.json [hunk 1]
```diff
@@ -142,6 +142,16 @@
                                         "providerName": "registrationapp",
                                         "fragmentId": "field/fingerprintM2sys"
                                     }
+                                },
+                                {
+                                    "type": "nationalPatientIdentifier",
```
Comment (pgesek) [inline omod/src/main/resources/apps/isanteplus_registration_app.json]: Is this correct? won't this result in the widget for scanning being shown twice on the page?

Comment (pawelcieszko) [reply omod/src/main/resources/apps/isanteplus_registration_app.json]: oops, yes I will undo it

**Comment by pawelcieszko** [comment]
The module has been started using an existing instance of OpenMRS and a fresh one. 
1. Existing instance (before the implementation): the parameters didn't change
2. Fresh one: Configured parameters has been added to global_property table containing specified values. The parameters with proprer values are displayed in openmrs/admin -> Advanced Settings. As far as I'm concerned these values are installed by metadata bundle.

**Comment by pgesek** [comment]
@pawelcieszko is the identifier type created? I am not talking about global properties, but the identifier type entity (admin -> identifier types)

**Comment by pawelcieszko** [comment]
No, the identifier type is not created. It seems that other modules have influence of generation because I haven't found any part of code in this module responsible for it (after setting fresh module there were 3 identifier types only at the beginning)
image

**Comment by pawelcieszko** [comment]
In this repo patient identifier types are explicitly initialized for some of them only:

https://github.com/IsantePlus/openmrs-module-isanteplus/blob/fdf15c1f949b5e0502a62f9173d380f89ac23e52/liquibase.xml#L688

**Comment by pawelcieszko** [comment]
@pgesek I have created the migration according to your suggestion. I have filled column values basing on existing biometric person identifier type:
image

The migration results in adding new Patient Identifier Type in admin -> Manage Identifier Types:
image

**Pull request merged**